packages/token-lists/src/buildList.ts

```diff
@@ -4,24 +4,33 @@ import { TokenList } from "@uniswap/token-lists";
 import { version } from "../package.json";
 import pancakeswapDefault from "./tokens/pancakeswap-default.json";
 import pancakeswapExtended from "./tokens/pancakeswap-extended.json";
+import pancakeswapTop100 from "./tokens/pancakeswap-top-100.json";
 
 const lists = {
   "pancakeswap-default": {
     list: pancakeswapDefault,
-    name: "PancakeSwap Default Token List",
+    name: "PancakeSwap Default",
     keywords: ["pancakeswap", "default"],
     logoURI:
       "https://assets.trustwalletapp.com/blockchains/smartchain/assets/0x0E09FaBB73Bd3Ade0a17ECC321fD13a19e81cE82/logo.png",
     sort: false,
   },
   "pancakeswap-extended": {
     list: pancakeswapExtended,
-    name: "PancakeSwap Extended Token List",
+    name: "PancakeSwap Extended",
     keywords: ["pancakeswap", "extended"],
     logoURI:
       "https://assets.trustwalletapp.com/blockchains/smartchain/assets/0x0E09FaBB73Bd3Ade0a17ECC321fD13a19e81cE82/logo.png",
     sort: true,
   },
+  "pancakeswap-top-100": {
```

Comment:
This is good, you'r getting there 👍 
Need to do similar modification in `checksum.ts` (just import list and add to the `lists`) and `default.test.ts` (add `pancakeswap-top-100` in [here](https://github.com/pancakeswap/pancake-toolkit/blob/master/packages/token-lists/test/default.test.ts#L48).
+ also make new command in package.json for top100 similar to other ones, read more in readme
https://github.com/pancakeswap/pancake-toolkit/tree/master/packages/token-lists#how-to-add-new-lists-within-this-repository